README.adoc

```diff
@@ -379,6 +582,12 @@ Tests run: 4, Failures: 0, Errors: 0, Skipped: 0
 
 == Tearing down the environment
 
+// Cloud-hosted guide instruction
+ifdef::cloud-hosted[]
+Go back to the terminals where you started the port forwarding the **system** service and 
```

Comment:
Go back to the terminals where you started the port forwarding the **system** service and 
the **inventory** service.
---->
Go back to the terminals where you started the port forwarding for the **system** service and 
the **inventory** service.